src/views/course-edit.vue

```diff
@@ -68,6 +66,7 @@ Dependencies:
       <!--<course-stats></course-stats>-->
 
       <course-preferences @settingsChanged="storeCourse"></course-preferences>
+      <course-feedback-download @success="$bvToast.show('author-toast')"></course-feedback-download>
```

Comment:
maybe a toast with a distinct text such as "Download started" or similar would be more suitable

Reply:
could be added on a later date, for now I would like to keep it this way so no one gets surprised by a sudden download and fears any malicious content

Reply:
But it does start a download?!